student/feedback.php

```diff
@@ -14,10 +14,11 @@
 // You should have received a copy of the GNU General Public License
 // along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
 
-/*
+/**
```

Comment:
/**
but
*/
doesn't seem consistent

is that right?

Reply:
Yes, that is actually right. (See any other file: this comment block starts with "/**)

Reply:
I guess I meant should it close with **/ to match but yeah I can see core is asymmetrical too